src/app/components/Headings/index.jsx

```diff
@@ -1,10 +1,18 @@
 import styled from 'styled-components';
-import { C_EBON, FF_NEWS_SANS_REG } from '../../lib/constants/styles';
+import {
+  C_EBON,
+  C_STORM,
+  FF_NEWS_SANS_REG,
+  GEL_SPACING_DBL,
+  GEL_SPACING_QUAD,
+} from '../../lib/constants/styles';
 import mediaQuery from '../../helpers/mediaQueries';
 
 export const Headline = styled.h1`
   color: ${C_EBON};
   font-family: ${FF_NEWS_SANS_REG};
+  margin: 0; // Reset
+  padding: ${GEL_SPACING_QUAD}px 0 ${GEL_SPACING_DBL}px 0;
```

Comment:
We should use `em` rather than `px`.

Reply:
Decided to implement as-is in this PR, and open up another PR to use `em` everywhere.